src/Pages/Questions/Questions.jsx

```diff
@@ -3,34 +3,34 @@ import "./Questions.css";
 import { useState } from "react";
 import { quizQuestion } from "./data";
 import { useQuiz } from "./../../Hooks/Context/quizContext";
-import {BsArrowRightCircleFill} from "react-icons/bs"
+import { BsArrowRightCircleFill } from "react-icons/bs";
+import { Link } from "react-router-dom";
 
```

Comment:
List the global imports first and local imports later. This improves readability.